examples/volumes/scaleio/README.md

```diff
@@ -80,7 +80,20 @@ $ kubectl create -f ./examples/volumes/scaleio/secret.yaml
 
 ## Deploying Pods with Persistent Volumes
 
-The following example shows how the ScaleIO volume plugin for Kubernetes automatically attach, format, and mount a volume for a deployed pod. This approach requires an existing ScaleIO volume.
+The example presented in this section shows how the ScaleIO volume plugin can automatically attach, format, and mount an existing ScaleIO volume for pod. 
+The Kubernetes ScaleIO volume spec supports the following attributes:
+
+| Attribute | Description |
+|-----------|-------------|
+| gateway | address to a ScaleIO API gateway (required)|
+| system  | the name of the ScaleIO system (required)|
+| protectionDomain| the name of the ScaleIO protection domain (default `default`)|
+| storagePool| the name of the volume storage pool (default `default`)|
+| storageMode| the storage provision mode: `ThinProvisionned` (default) or `ThickProvisionned`|
+| volumeName| the name of an existing volume in ScaleIO (required)|
+| secretRef:name| reference to a configuered Secret object (required, see Secret earlier)|
+| readOnly| specifies the access mode to the mounted volume (default `false`)|
+| fsType| the file system to use for the volume (default `xfs`)|
```

Comment:
(also for rest for the patch) this is inconsistent with api doc https://github.com/kubernetes/kubernetes/blob/master/pkg/api/types.go#L1091.